### PR TITLE
check if room is not full

### PR DESCRIPTION
**Pull request by Gregi96**

### CODE REVIEW

File: client/src/components/Room.tsx [hunk 1]
```diff
@@ -134,6 +170,10 @@ const AcceptContainer = styled.div`
 `
 
 const InviteContainer = styled.div`
+    position: absolute;
+    top: 0;
+    transform: translateX(-50%);
+    left: 50%;
```
Comment (gtarnopolski) [inline client/src/components/Room.tsx]: why did you apply the transform and left

File: client/src/lib/stores/useRoomStore.ts [hunk 2]
```diff
@@ -34,6 +37,14 @@ export const useRoomStore = (): useRoomStoreResponse => {
     const navigation = useNavigate()
     const [activeCamera, setActiveCamera] = useState(false)
     const [activeMicrophone, setActiveMicrophone] = useState(false)
+    const [usersInRoom, setUsersInRoom] = useState(0)
+    const [checkedUsersInRoom, setCheckedUsersInRoom] = useState(false)
+
+    const checkCountOfUsersInRoom = (roomId: string) => {
```
Comment (gtarnopolski) [inline client/src/lib/stores/useRoomStore.ts]: proxy

File: client/src/lib/stores/useRoomStore.ts [hunk 3]
```diff
@@ -18,7 +18,10 @@ export type useRoomStoreResponse = {
     stream?: MediaStream,
     activeCamera: boolean,
     activeMicrophone: boolean,
+    usersInRoom: number,
```
Comment (gtarnopolski) [inline client/src/lib/stores/useRoomStore.ts]: this `useRoomStoreResponse` is completely redundant

File: server/src/room/index.ts [hunk 4]
```diff
@@ -39,6 +43,14 @@ export const roomHandler = (socket: Socket) => {
         })
     }
 
+    const getUsers = ({ roomId }: GetUsersProps) => {
```
Comment (gtarnopolski) [inline server/src/room/index.ts]: proxy

File: client/src/components/Room.tsx (hunk 1 above)
Comment (Gregi96) [reply client/src/components/Room.tsx]: beeter:

top: 0
left: 50%,
transform: translateX(-50%)

I used this record to get centered element on x axis

File: client/src/lib/stores/useRoomStore.ts (hunk 3 above)
Comment (Gregi96) [reply client/src/lib/stores/useRoomStore.ts]: I need this because i listen "ws.on('get-users)" and next I set count of users in store state

**Pull request merged**